sdks/storage/examples/get_messages.rs

```diff
@@ -0,0 +1,36 @@
+#[macro_use]
+extern crate log;
+use azure_storage::core::prelude::*;
+use azure_storage::queue::prelude::*;
```

Comment:
This is a good example of why I think we should think about the storage crate's public API. It's a bit weird to have to export both core and queue's prelude modules.

Reply:
Yes it's because right now the *storage client* comes from the *storage core* crate and the *storage queue client* trait comes from the *storage queue* crate. 
I agree, some rearranging is overdue.